src/pflib/packing/SoftWrappedECONDEventPacket.cxx

```diff
@@ -0,0 +1,64 @@
+#include "pflib/packing/SoftWrappedECONDEventPacket.h"
+
+#include "pflib/packing/Hex.h"
+#include "pflib/packing/Mask.h"
+
+namespace pflib::packing {
+
+SoftWrappedECONDEventPacket::SoftWrappedECONDEventPacket(std::size_t n_links)
+    : data{n_links} {}
+
+void SoftWrappedECONDEventPacket::from(std::span<uint32_t> frame) {
+  /**
+   * The software emulation adds another header before the ECOND packet,
```

Comment:
@tomeichlersmith  - question, why does the software emulation add this header? Is this the header that the DAQ will add?

Reply:
The DAQ will be adding an extra header, I am unsure if this header has the same format. The good news is that the ECOND packet (here: `pflib::packing::ECONDEventPacket`) is always being wrapped. I'm keeping it that way on purpose since that structure is stable.